test/string.test.ts

```diff
@@ -51,6 +51,12 @@ describe("String", () => {
         })
     })
 
+    describe("reverse", () => {
```

Comment:
I'm not sure what's going on with the coverage. Even after adding this test and the other case below the report seems to think we're missing some coverage on string. :shrug: